data.sql

```diff
@@ -22,3 +22,20 @@ INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES('Blossom', '1998-10-13', 3, true, 17);
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES('Ditto', '2022-05-14', 4, true, 22);
+
+INSERT INTO owners(full_name, age)
+VALUES('Sam Smith', 34),('Jennifer Orwell', 19), ('Bob', 45), ('Melody Pond', 77),
+('Dean Winchester', 14), ('Jodie Whittaker', 38);
+
+INSERT INTO species(name)
+VALUES('Pokemon'), ('Digimon');
+
+
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 1 WHERE name NOT LIKE '%mon';
+
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon','Pikachu');
+UPDATE animals SET owner_id = 3 WHERE name in ('Devimon', 'Plantmon');
+UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
+UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon');
```

Comment:
- [ ] OPTIONAL
It's recommended your queries do not explicitly use the values of _id's_; Doing so will be impractical in the event of querying large databases, as we can't memorize all _id's_. We should rather do so using some given criteria, similar to:

```sql
# data.sql
 UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith')
   WHERE name = 'Agumon';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Jennifer Orwell')
   WHERE name = 'Gabumon' OR name = 'Pikachu';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Bob')
   WHERE name = 'Devimon' OR name = 'Plantmon';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Melody Pond')
   WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Dean Winchester')
   WHERE name = 'Angemon'  OR name = 'Boarmon';
```